charts/netbox/templates/cronjob.yaml

```diff
@@ -85,9 +85,11 @@ spec:
             - name: config
               mountPath: /run/config/netbox
               readOnly: true
+            {{- if (eq .Values.customSecretMount false) }}
```

Comment:
```suggestion
            {{- if not .Values.customSecretMount }}
```

Should be sufficient

Reply:
That is better, adjusted in [a8fe223](https://github.com/netbox-community/netbox-chart/pull/576/commits/a8fe22364c71915e2a49d28cff20faa58631a1cc)